src/api/general-services/pipeline-manage/constructors/create-new-job-if-not-exist.js

```diff
@@ -0,0 +1,78 @@
+const config = require('../../../../config');
```

Comment:
One of the "constructors" for the pipeline. Depending on the environment, a different task gets put in the skeleton.